float32 steering_angle